build.sbt

```diff
@@ -13,14 +13,24 @@ import src.main.scala.licenses.{
   SBTDistributionComponent
 }
 
-import scala.sys.process._
+import scala.sys.process.*
+import Dependencies.*
+import JarExtractor.{
+  CopyToOutputJar,
+  LinuxX86_64,
+  MacOSArm64,
+  MacOSX86_64,
```

Comment:
`MacIntel64` - maybe. Or anything less cryptic than OSX86_64 ...

Reply:
Renamed `x86_64` to `AMD64` in 8470150a37d1cef9fee9b8b2791c9f8ce9346290